src/components/RawDisplay.tsx

```diff
@@ -15,16 +16,23 @@ export default function RawDisplay(props: {
   const fluentStyle = fluentStyles();
   const { t } = useTranslation();
 
-  return (
+  const isLocalDevMode = useMemo(
+    () => process.env.NODE_ENV === "development",
+    [process.env.NODE_ENV]
+  );
+
+  const rawDisplay = (
     <Accordion collapsible>
       <AccordionItem value="raw">
         <AccordionHeader className={fluentStyle.accordionHeader}>
           {t("debug.showResponse")}
         </AccordionHeader>
-        <AccordionPanel className="rawResponseDetail">
+        <AccordionPanel className="rawResponseDetails">
           <pre>{`${JSON.stringify(props.data, null, 1)}`}</pre>
         </AccordionPanel>
       </AccordionItem>
     </Accordion>
   );
+
+  return isLocalDevMode ? rawDisplay : <> {}</>;
```

Comment:
I'm unsure about hiding this in dev env only
A. we can find and do basic diagnostics on the go, in real-life situations
B. in the spirit of open data, anyone can get a simple glimpse into the publicly available data

Reply:
> A. we can find and do basic diagnostics on the go, in real-life situations

Technically there is no difference between the local dev branch and the production branch as they are fetching the same API for data, so we can always use the local dev branch for debugging. ( That's why most large industrial-grade projects will have `UAT` and `STG` for debugging, as they don't want the users to see rawResponse)

> B. in the spirit of open data, anyone can get a simple glimpse into the publicly available data

UI/UX-wise, showing rawData to users is not a good approach as rawData has no usage for them most of the time, some may find it interesting at the very beginning, but later on, they will just ignore it and go straight to the bookmarkedItems for arrivalTimes. 

To keep the UI clean and simple yet precise, we should avoid showing data that has no functional usage to users